pkg/build/info.proto

```diff
@@ -25,7 +25,7 @@ message Info {
   optional string go_version = 1 [(gogoproto.nullable) = false];
   optional string tag = 2 [(gogoproto.nullable) = false];
   optional string time = 3 [(gogoproto.nullable) = false];
-  optional string dependencies = 4 [(gogoproto.nullable) = false];
+  optional string revision = 4 [(gogoproto.nullable) = false];
```

Comment:
Indeed, this proto is special in that it is only really used for node->UI communication. I'll change the tag if I need to eat a CI cycle for some other reason.